runtime/compiler/control/JITaaSCompilationThread.cpp

```diff
@@ -2562,7 +2564,63 @@ remoteCompilationEnd(
             returnCode = compilationAotRelocationInterrupted;
             }
 
-         if (!relocatedMetaData)
+         if (relocatedMetaData) 
+            {
+            if (TR::Options::getVerboseOption(TR_VerboseJITaaS))
+               {
+               TR_VerboseLog::writeLineLocked(TR_Vlog_JITaaS, "JITaaS Client successfully relocated metadata for %s", comp->signature());
+               }
+
+            if (J9_EVENT_IS_HOOKED(jitConfig->javaVM->hookInterface, J9HOOK_VM_DYNAMIC_CODE_LOAD))
```

Comment:
Since this part of the code is identical to the one use for non-JITaaS compilations can we refactor it into a separate function that is going to be called in two different places?